src/components/comments/Answer.js

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+
+export default function Answer(props) {
+  return (
+    <div style={styles.answerContainer}>
+      <div style={styles.answerHeaderStyle}>{props.answerHeaderText}</div>
+      <div style={styles.answerStyle}>{props.answerText}</div>
+    </div>
+  );
+}
+
+const styles = {
+  answerContainer: {
+    backgroundColor: "#EEF0F1",
+    borderRadius: "25px",
+    marginLeft: "200px",
+    marginRight: "10px",
+    padding: "10px",
+  },
+  answerHeaderStyle: {
+    textAlign: "left",
+    color: "black",
+    fontFamily: "Mulish",
+    fontStyle: "normal",
```

Comment:
This is by default, so you don't need to set it! (you do it elsewhere as well, you can remove those)